lib/parser.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:stomp_dart_client/stomp_frame.dart';
 
 abstract class Parser {
-  bool escapeHeaders;
+  late bool escapeHeaders;
```

Comment:
Right, the sock_js_parser is not using it, but the flag is dynamically injected in the `_onConnectFrame` handler, after one of both parsers is already instantiated:

```dart
if (frame.headers['version'] != '1.0') {
  _parser.escapeHeaders = true;
} else {
  _parser.escapeHeaders = false;
}
```

Therefore, the only easy way around this I see right now would be an explicit `is StompParser` check, but this is probably brittle too. I'd tend to leave it as is. Another option would be to drop support for the 1.0 protocol, since that is the only reason this is even needed (I think?), but I guess we would not want that.

Reply:
Okay. Did not see this. We leave it then.